src/network.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+import { ImagesURLs, Theme } from './interfaces'
+
+const BASE_URL = 'https://gateway-dev.arcana.network/api/'
+
+function getImageUrls(appId: string, theme: Theme): ImagesURLs {
```

Comment:
Inline the return type.

```suggestion
function getImageUrls(appId: string, theme: Theme): { horizontal: string, vertical: string } {
```